client/js/bundles/procedure/administrationProcedureTypeEdit.js

```diff
@@ -15,15 +15,13 @@
  * import DpAccordion from '@DpJs/components/core/DpAccordion'
  * import DpEditor from '@DpJs/components/core/DpEditor/DpEditor'
  */
+import { defineAsyncComponent } from 'vue'
 import { dpValidate } from '@demos-europe/demosplan-ui'
 import { initialize } from '@DpJs/InitVue'
 import ProcedureTypeSelect from '@DpJs/components/procedure/admin/ProcedureTypeSelect'
 
 const components = {
-  DpEditor: async () => {
-    const { DpEditor } = await import('@demos-europe/demosplan-ui')
-    return DpEditor
-  },
+  DpEditor: defineAsyncComponent(() => import('@demos-europe/demosplan-ui')),
```

Comment:
Are you sure that this works? Shouldn't you import DpEditor specifically?

Reply:
yes, done in https://github.com/demos-europe/demosplan-core/pull/2424/commits/160ca655c6617d9427f3aea0e97f0b87c8b4b677, thnx!

Reply:
Thanks
Done in https://github.com/demos-europe/demosplan-core/pull/2424/commits/3dbc0f8e5ea38fb9d49214443c10cf6b699b6046